lib/modules/datasource/unity3d/schema.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod';
+
+const UnityReleaseNote = z.object({
+  url: z.string(),
+});
+
+const UnityRelease = z.object({
+  version: z.string(),
+  releaseDate: z.string(),
+  releaseNotes: UnityReleaseNote,
```

Comment:
are you sure this object always exists for all packages / streams?

Reply:
I followed the instructions in the official API documentation: https://services.docs.unity.com/release/v1/#tag/Release/operation/getUnityReleases